packages/api/src/utils/env.js

```diff
@@ -0,0 +1,3 @@
+export function magicTestModeFromEnv (env) {
+  return Boolean(env?.NEXT_PUBLIC_MAGIC_TESTMODE_ENABLED)
```

Comment:
**danger danger**
```node
> Boolean("false")
true
```

Reply:
@olizilla nice. I actually thought of this too when I authored it :) Usually what I do to accomodate for this is `JSON.parse` the string, which will parse this case to `false`. I'll do that and add a test case

Reply:
I made it so providing `"false"` results in testMode being disabled